book/whats_new/aug-18/swift-5.0-migration.md

```diff
@@ -0,0 +1,109 @@
+    #else
+    <contents of original iOS/Framework.framework/Framework-Swift.h>
+    #endif
+    ```
+
+## Internal Applicaster Plugin Migration
+
+The following instructions are an for Internal Applicaster plugins that creates a closed frameworks.
+
+1. Install cocoapods version 1.7.5 using the following command:
+    ```
+    sudo gem install cocoapods
+    ```
+
+2. Upgrade the following gems inside the Gemfile:
+    ```
+    gem 'xcpretty'
+    gem 'cocoapods', '= 1.7.5'
+    gem 'fastlane' (remove the versioning)
+    ```
+
+3. Update the Gemfile.lock by running `bundle install` in your main plugin directory.
+
+4. Go to circle-ci confog.yml file and change the following:
+    * Change the xcode version to 10.2.1
+    * Change the deploy version to 0.8.11
+    * Increment the caching key to a new version, for example: 
+        ```
+        KanCentralizedViewsPlugin-v3-{<other_values>} 
+        ```
+        Should changed to:
+        
+        ```
+        KanCentralizedViewsPlugin-v4-{<other_values>}.
+        ```
+    * Make sure that all caching key’s have the same prefix name.
+    * Cache the pod directory by adding to circle script the following `Restore Pods cache` and `save_cache` caching phases (check Zapp-iOS project as an example):
+5. Update podspec dependencies to support swift 5.0, please check the following:
+    * Make sure you also update the podspec files located inside the /Podspec-templates directory.
+    * Inside each podspec file, make sure that the s.xcconfig.'SWIFT_VERSION' value is set to 5.0.
+    * Update all dependencies to support swift 5.0 specs.
+6. Update podfile structure according to the following steps:
+    * dependencies to support swift 5.0 specs.
+    * Use the new cocoapods CDN source:
+        * Remove `source 'git@github.com:CocoaPods/Specs.git'`.
+        * Add the new CDN `source  'https://cdn.cocoapods.org/'` (If you have private specs, add the new source below all of the other sources).
+        * Run `pod deintegrate` to deintegrate CocoaPods from your project. The move to the latest CDN source may cause problems, therefore, it’s recommended to clean all cocoapods related files.
+        * Run `pod install`.
+7. Run pod update command and open the project with Xcode 10.2.1.
+8. In side Xcode, in each of your TARGETS, change the “swift language version” to swift 5.
+9. Build the framework in Xcode and make sure it’s working without errors.
+10. Increment the plist framework version to a major change version notion.
+11. Push your code with GIT and make sure circle-ci build process ended successfully.
+12. Verify that a new podspec file was created inside our private [CocoaPods-Private](https://github.com/applicaster/CocoaPods-Private) or public [CocoaPods](https://github.com/applicaster/CocoaPods) repositories.
+13. Create a new manifest file that correlates to the same version number in the framework plist and upload to Zapp.
+
+## 3rd Party Plugins Migration
```

Comment:
I feel like both guides should also specify Xcode version to use

Reply:
added that